packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -24,7 +24,7 @@ export function AriaLiveRegion(regionName: string) {
 
 export interface FocusTargetController {
   setTarget(element: HTMLElement | undefined): void;
-  focusAfterSearch(): void;
+  focusAfterSearch(): Promise<void>;
```

Comment:
It made sense to me that `focusAfterSearch` should return a promise that resolves after something was focused. If you disagree, I can find a different solution.